src/SSVOpenHexagon/Core/HGUpdate.cpp

```diff
@@ -27,9 +27,18 @@ void HexagonGame::update(ssvu::FT mFT)
 
     // TODO: refactor to avoid repetition, and truncate floating point number
     // TODO: also show best record (here) and last run + best record (in menu)
-    steamManager.set_rich_presence_in_game(
-        levelData->name, status.getTimeSeconds());
-    steamManager.run_callbacks();
+
+    const int DELAY_TO_UPDATE = 5; // X seconds
```

Comment:
```suggestion
    constexpr float DELAY_TO_UPDATE = 5; // X seconds
```

Make this `constexpr` (minor change, it means it's a compile-time constant) and `float` so it matches the type we're trying to compare against.